src/PostgREST/SchemaCache.hs

```diff
@@ -652,130 +645,65 @@ tablesSqlQuery =
               information_schema._pg_truetypid(a.*, t.*),
               information_schema._pg_truetypmod(a.*, t.*)
           )::integer AS character_maximum_length,
-          COALESCE(bt.typname, t.typname)::name AS udt_name,
+          COALESCE(bt.oid, t.oid) AS base_type,
           a.attnum::integer AS position
       FROM pg_attribute a
           LEFT JOIN pg_description AS d
               ON d.objoid = a.attrelid and d.objsubid = a.attnum
           LEFT JOIN pg_attrdef ad
               ON a.attrelid = ad.adrelid AND a.attnum = ad.adnum
-          JOIN (pg_class c JOIN pg_namespace nc ON c.relnamespace = nc.oid)
+          JOIN pg_class c
               ON a.attrelid = c.oid
-          JOIN (pg_type t JOIN pg_namespace nt ON t.typnamespace = nt.oid)
+          JOIN pg_type t
               ON a.atttypid = t.oid
-          LEFT JOIN (pg_type bt JOIN pg_namespace nbt ON bt.typnamespace = nbt.oid)
+          LEFT JOIN pg_type bt
               ON t.typtype = 'd' AND t.typbasetype = bt.oid
-          LEFT JOIN (pg_collation co JOIN pg_namespace nco ON co.collnamespace = nco.oid)
-              ON a.attcollation = co.oid AND (nco.nspname <> 'pg_catalog'::name OR co.collname <> 'default'::name)
-          LEFT JOIN pg_depend dep
-              ON dep.refobjid = a.attrelid and dep.refobjsubid = a.attnum and dep.deptype = 'i'
-          LEFT JOIN pg_class seqclass
-              ON seqclass.oid = dep.objid
-          LEFT JOIN pg_namespace seqsch
-              ON seqsch.oid = seqclass.relnamespace
+          LEFT JOIN pg_collation co
+              ON a.attcollation = co.oid AND (co.collnamespace <> 'pg_catalog'::regnamespace OR co.collname <> 'default'::name)
```

Comment:
Nice! The query looks much simpler now.

I also had a doubt about this LEFT JOIN before, I don't see that the values are used anywhere in the query, should it be safe to remove it?

```suggestion
```